src-tauri/src/mod_management/mod.rs

```diff
@@ -31,3 +35,96 @@ pub fn set_mod_enabled_status(
 
     Ok(())
 }
+
+
+/// Parses `mod.json` for mod name
+// TODO: Maybe pass PathBuf or serde json object
```

Comment:
Still TODO?

Reply:
Yup but not for this PR IMO. This would basically just give negligible performance increase and slightly better looking code but I don't have enough experience with `PathBuf` object yet to tackle this in this PR.